beacon-chain/sync/subscriber_beacon_attestation.go

```diff
@@ -27,15 +28,18 @@ func (s *Service) committeeIndexBeaconAttestationSubscriber(_ context.Context, m
 	}
 	s.setSeenCommitteeIndicesSlot(data.Slot, data.CommitteeIndex, a.GetAggregationBits())
 
-	exists, err := s.cfg.attPool.HasAggregatedAttestation(a)
-	if err != nil {
-		return errors.Wrap(err, "could not determine if attestation pool has this attestation")
+	if features.Get().EnableExperimentalAttestationPool {
+		return s.cfg.attestationCache.Add(a)
```

Comment:
It doesn't make sense to perform a redundancy check here (similarly to how the old design calls `HasAggregateAttestation`) because this function deals with unaggregated atts.